mne/utils.py

```diff
@@ -1205,6 +1205,7 @@ def set_memmap_min_size(memmap_min_size):
     'MNE_SKIP_TESTING_DATASET_TESTS',
     'MNE_STIM_CHANNEL',
     'MNE_USE_CUDA',
+    'MNE_TESTING',
```

Comment:
The calls to freesurfer here aren't actually that slow. It is the conversion of flash images that took a long time before. Now we have them precomputed.